schemas/common/orgunit.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/orgunit",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "object",
+  "meta:extensible": true,
+  "title": "Org Unit",
+  "description": "The organisation unit of a parent organisation.",
```

Comment:
Should be "organi*z*ation", US spelling.

Reply:
done!